### ISSUE TITLE
Merging an empty splitter subview causes an exception

### ISSUE THREAD

**Issue by Prism019** [EDDiscovery/EDDiscovery#2530]
```
There was an unhandled UI exception.

==== BEGIN ====
System.ArgumentOutOfRangeException: Index 0 is out of range.
Parameter name: index
   at System.Windows.Forms.Control.ControlCollection.get_Item(Int32 index)
   at ControlHelpersStaticFunc.Merge(SplitContainer currentsplitter, Int32 panel) in C:\Code\EDDiscovery\BaseUtilities\BaseUtilities\WinForms\ControlHelpers.cs:line 463
   at EDDiscovery.UserControls.UserControlContainerSplitter.toolStripMergePanel2_Click(Object sender, EventArgs e) in C:\Code\EDDiscovery\EDDiscovery\UserControls\GridsSplitters\UserControlContainerSplitter.cs:line 424
   at System.Windows.Forms.ToolStripItem.RaiseEvent(Object key, EventArgs e)
   at System.Windows.Forms.ToolStripMenuItem.OnClick(EventArgs e)
   at System.Windows.Forms.ToolStripItem.HandleClick(EventArgs e)
   at System.Windows.Forms.ToolStripItem.HandleMouseUp(MouseEventArgs e)
   at System.Windows.Forms.ToolStripItem.FireEventInteractive(EventArgs e, ToolStripItemEventType met)
   at System.Windows.Forms.ToolStripItem.FireEvent(EventArgs e, ToolStripItemEventType met)
   at System.Windows.Forms.ToolStrip.OnMouseUp(MouseEventArgs mea)
   at System.Windows.Forms.ToolStripDropDown.OnMouseUp(MouseEventArgs mea)
   at System.Windows.Forms.Control.WmMouseUp(Message& m, MouseButtons button, Int32 clicks)
   at System.Windows.Forms.Control.WndProc(Message& m)
   at System.Windows.Forms.ScrollableControl.WndProc(Message& m)
   at System.Windows.Forms.ToolStrip.WndProc(Message& m)
   at System.Windows.Forms.ToolStripDropDown.WndProc(Message& m)
   at System.Windows.Forms.Control.ControlNativeWindow.OnMessage(Message& m)
   at System.Windows.Forms.Control.ControlNativeWindow.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)
===== END =====
```

How to reproduce:
- Create a splitter tab.
- Split a side.
- Set one of the panels of the split side to a splitter.
- Merge the side.

**Comment by robbyxp1** [comment]
Ack, debugging

**Comment by robbyxp1** [comment]
Fixed, 11.0, thanks for the report.
